### ISSUE TITLE
Disable Google Account access for certain apps

### ISSUE THREAD

**Issue by moodl** [microg/android_packages_apps_GmsCore#235]
It would be nice to have an option for disabling account access for apps. For example I don't want Google Maps to know who I am. Sorry for my bad english.

**Comment by Primokorn** [comment]
You can use XPrivacy to restrict this access.

**Comment by moodl** [comment]
@Primokorn Right, but a separate feature would still be nice. Also some people don't want to use Xposed and it isn't available for Nougat yet.

**Comment by mar-v-in** [comment]
1. microG uses Android's account storage to store accounts. All apps with GET_ACCOUNTS permission can read them (on Android 6.0+ this is bound to the Contacts permission group). There is nothing we can do against this, the account manager can not be configured per app.
2. Certain Google apps have access to the microG internal account database via the Google People API. We currently trust blindly all Google apps (by signature) to be allowed to access this database. We should probably make it accessibly only after confirmation (or if GET_ACCOUNTS is granted anyway).

**Comment by ale5000-git** [comment]
For some apps you can hide the accounts with:
`pm revoke com.name.app android.permission.GET_ACCOUNTS`
